.github/workflows/ci.yml

```diff
@@ -646,14 +646,32 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: 11
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            product-tests:
+              - 'testing/trino-product-tests*/**'
+              - 'testing/trino-testing-services/**'
+              - 'core/trino-server/**'
```

Comment:
Why trino-server? Is it to catch new connectors?

Reply:
`trino-server` is the only module that does not properly define dependencies. It relies on Provisio to find artifacts. So if anything changes in this module, GIB wouldn't properly include all plugins on its own.

Reply:
Can you add a short comment that it's there because of no explicit dependencies in pom?

Reply:
Done!